compiler/test/compilable/test21177.d

```diff
@@ -27,48 +33,38 @@ void main()
         #line 100
         printf("%m this is a string in errno");
         printf("%s %m", "str".ptr, 2);
```

Comment:
Should the chkformat complain when there are more arguments than format specifiers @WalterBright?